### PR TITLE
Reverts change to 'get server side props'

### PR DESCRIPTION
**Pull request by andycarrell**

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -2,6 +2,9 @@
   command = "npm run build"
   publish = "out"
 
+[functions]
+  included_files = ["data/*.txt"]
```
Comment (andycarrell) [inline netlify.toml:6]: Well, this is about to be inaccurate ;)

Comment (theramis) [reply netlify.toml:6]: hehe sorrrrry

**Comment by theramis** [comment]
Nice work dude!

Kinda annoyed at myself the fix was so simple in the end 😅

File: netlify.toml (hunk 1 above)
Comment (theramis) [reply netlify.toml:6]:
```suggestion
  included_files = ["data/vaccineData/*.txt"]
```

**Comment by jishaal** [comment]
> fix was so simple in the end 

Fix for... 👀 

Also sad this brings back the default 404 page on the previews 😢

**Comment by andycarrell** [comment]
Looks like that didn't work :( afk but will take a look when I can

**Review by theramis — APPROVED** [review]

**Comment by theramis** [comment]
> > fix was so simple in the end
> 
> Fix for... 👀

To get SSR working with the data files. I couldn't figure out to package them originally 😅

**Comment by jishaal** [comment]
Ohhh! Noice noice

**Review by jishaal — APPROVED** [review]

**Pull request merged**